docs/getting-started/security-ui.asciidoc

```diff
@@ -32,6 +32,20 @@ The navigation menu contains direct links and expandable groups, identified by t
 [role="screenshot"]
 image::images/nav-overview.gif[Overview of the navigation menu]
 
+[[visualization-actions]]
+[float]
+== Visualization actions
+
+Many {elastic-sec} histograms, graphs, and tables display an *Inspect* button (image:images/inspect-icon.png[Inspect icon,19,19]) when you hover over them, so you can examine the {es} queries used to retrieve data throughout the app. 
```

Comment:
Optional edit to make this more actionable.

```suggestion
Many {elastic-sec} histograms, graphs, and tables display an *Inspect* button (image:images/inspect-icon.png[Inspect icon,19,19]) when you hover over them. Click the *Inspect* button to examine the {es} queries used to retrieve data throughout the app. 
```

Reply:
```suggestion
Many {elastic-sec} histograms, graphs, and tables display an *Inspect* button (image:images/inspect-icon.png[Inspect icon,19,19]) when you hover over them. Click to examine the {es} queries used to retrieve data throughout the app. 
```